accepted/00000-auto-generate-bootstrap-repositories.md

```diff
@@ -0,0 +1,71 @@
+- Feature Name: Autogenerate Bootstrap Repositories
+- Start Date: 2020-02-19
+- RFC PR: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Automatically generate bootstrap repositories
+
+# Motivation
+[motivation]: #motivation
+
+To be able to register a system to Uyuni Server you need to have the software installed which provide the required tooling.
+
+A bootstrap repository provide the software for the supported operating systems with its dependencies. The content for the
+bootstrap repositories come from the normal channels which must be synchronized and updated on regular base.
+
+Users very often do not know this, or forget to create them when they synchronize a new product. Also the bootstrap repository
+is only updated when the user execute the generator script again.
+
+This RFC should describe how to change the current tools to automatically 
+execute the generation of bootstrap repositories after
+a new product was added and after a channel changed when a maintenance update was synced to Uyuni Server.
+
+# Detailed design
+[design]: #detailed-design
+
+The currently exiting tool `mgr-create-bootstrap-repo` list all available bootstrap repositories it could generate.
+This list depends on the products the user has synchronized.
```

Comment:
Not sure if I'd use the word `products` here. Sounds too much like `SUSE Products`.

The list will also include `Debian` soon, or it includes `Ubuntu for Uyuni`.

Reply:
Tell me a better word. OSes?

Reply:
That's a good question :-\

We can keep `products` as long we  don't think about `SUSE Products from SCC` during the implementation :-)

Maybe a not to explain what `product` at the start of the RFC would help:

> ** NOTE: When talking about "products" at this RFC, we are talking about either products from SUSE SCC, or other OS added manually by the user or with `spacewalk-common-channels` such as Ubuntu, CentOS, Debian, etc.**

Reply:
I used products/channels now.